src/main/java/org/parabot/randoms/oswar/ItemPicker.java

```diff
@@ -0,0 +1,66 @@
+package org.parabot.randoms.oswar;
+
+import org.json.simple.JSONObject;
+import org.parabot.core.Core;
+import org.parabot.environment.api.utils.Time;
+import org.parabot.environment.api.utils.WebUtil;
+import org.parabot.environment.randoms.Random;
+import org.parabot.environment.randoms.RandomType;
+import org.rev317.min.api.methods.Game;
+import org.rev317.min.api.methods.Interfaces;
+import org.rev317.min.api.methods.Menu;
+
+
+/**
+ * @author AlexanderBielen
+ */
+public class ItemPicker implements Random {
+    private final int INTERFACE_ID = 33300;
+    private final String URL = "http://bdn.parabot.org/api/v2/data/items/";
```

Comment:
Check out [this API](https://github.com/Parabot/Parabot-317-API-Minified/blob/db3b9620eeb55e66686deccfb8b86df59fd83b38/src/main/java/org/rev317/min/api/methods/Items.java). You might be able to use it instead of re-code it :)